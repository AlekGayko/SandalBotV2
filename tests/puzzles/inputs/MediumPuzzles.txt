5000 r7/6p1/6pk/4Q1N1/6pK/5N2/8/1b6 w - - 0 1
5000 B3b3/2p5/P1Rp4/PK1k1P2/3p1P2/3P4/1R6/8 w - - 0 1
5000 8/7p/8/8/3p4/3P2RR/6PP/5K1k w - - 0 1
5000 k7/1q6/2N5/2NNN3/8/8/1K5R/8 w - - 0 1
5000 8/8/4B3/R6r/p1p1pr2/3kP3/3P1P2/3K3N w - - 0 1
5000 2kq1r2/p7/2p5/1pR5/1K1P1p2/1PP2R2/P2r4/5Q2 b - - 0 1
5000 k1K5/p7/8/2n5/8/8/7B/R7 w - - 0 1