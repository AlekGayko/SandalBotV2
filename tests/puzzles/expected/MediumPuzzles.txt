e5b8 M3
b5a4 M2
h3h5 M3
c5b7 M4
e6h3 M2
d8a5 M3
h2g1 M3